src/Eloquent/Builder.php

```diff
@@ -228,16 +233,15 @@ protected function ensureOrderForCursorPagination($shouldReverse = false)
         }
 
         if ($shouldReverse) {
-            $this->query->orders = collect($this->query->orders)->map(function ($direction) {
-                return $direction === 1 ? -1 : 1;
-            })->toArray();
+            $this->query->orders = collect($this->query->orders)
+                ->map(fn (int $direction) => $direction === 1 ? -1 : 1)
```

Comment:
Good use of an arrow function.
```suggestion
                ->map(static fn (int $direction) => $direction === 1 ? -1 : 1)
```